yaml-tests/yaml-tests.gradle

```diff
@@ -128,8 +128,16 @@ task downloadExternalServer(type: Copy) {
     into project.layout.buildDirectory.dir('externalServer')
 }
 
+task buildLocalServer(type: Copy) {
+    dependsOn ":fdb-relational-server:package", "downloadExternalServer"
```

Comment:
Why does this depend on `downeladExternalServer`?
Should it depend on `cleanExternalServerDirectory` instead?

Reply:
I was trying to avoid a potential race condition where the cleanup happened after one of the defendants. I think this new task is better arranged.

Reply:
The race condition is not possible, gradle is creating a dependency graph, and will only run `cleanExternalServerDirectory` once.
That being said, the new code is cleaner.